client/app/expenses/page.tsx

```diff
@@ -20,6 +20,7 @@ export default function ExpensePage() {
             variant={'outline'}
             className='text-base md:text-lg'
             onClick={() => router.push('/expenses/create')}
+            aria-label='Visit create expense item page'
```

Comment:
In this place, you don't need to add an aria label because the button already has the string `Add`! The following changes are same

Reply:
I referred to your commit for fixing bugs. I thought this would help with accessibility. Do you still want me to remove them?
<img width="943" alt="Screenshot 2024-04-18 at 13 07 33" src="https://github.com/HAK2024/fairy-share/assets/132785671/b127aef7-505f-4ead-8d33-23ed70fbde18">
https://github.com/HAK2024/fairy-share/pull/123/files

Reply:
Could you delete the aria label for all changes you made?

Reply:
I removed them, including the one above. XD
117b6c61359734e8d183e5c2df0628404f71c777